src/constants/LayoutConstants.js

```diff
@@ -0,0 +1,114 @@
+export const CARD_SIZES = {
+  XSMALL: 'XSMALL',
+  SMALL: 'SMALL',
+  TALL: 'TALL',
+  MEDIUM: 'MEDIUM',
+  WIDE: 'WIDE',
+  LARGE: 'LARGE',
+  XLARGE: 'XLARGE',
+};
+
+export const CARD_TYPES = {
+  LINE: 'LINE',
+  VALUE: 'VALUE',
+  TABLE: 'TABLE',
+};
+
+export const DASHBOARD_SIZES = {
+  XLARGE: 'xl',
+  LARGE: 'lg',
+  MEDIUM: 'md',
+  SMALL: 'sm',
+  XSMALL: 'xs',
+};
+
+export const DASHBOARD_COLUMNS = {
+  xl: 12,
+  lg: 12,
+  md: 12,
+  sm: 6,
+  xs: 4,
+};
+
+export const DASHBOARD_BREAKPOINTS = {
+  xl: 1800,
```

Comment:
Should these breakpoints match the Carbon 10 design?

https://www.carbondesignsystem.com/guidelines/layout#breakpoints